x/mint/minter_test.go

```diff
@@ -44,10 +44,38 @@ func TestNextInflation(t *testing.T) {
 	for i, tc := range tests {
 		minter.Inflation = tc.setInflation
 
-		inflation := minter.NextInflation(params, tc.bondedRatio)
+		inflation := minter.NextInflationRate(params, tc.bondedRatio)
 		diffInflation := inflation.Sub(tc.setInflation)
 
 		require.True(t, diffInflation.Equal(tc.expChange),
 			"Test Index: %v\nDiff:  %v\nExpected: %v\n", i, diffInflation, tc.expChange)
 	}
 }
+
+func TestBlockProvision(t *testing.T) {
+	minter := InitialMinter(sdk.NewDecWithPrec(1, 1))
+	params := DefaultParams()
+
+	secondsPerYear := int64(60 * 60 * 24 * 365)
```

Comment:
There are `365.25` days per year, on average.

Reply:
Can we just subtract time and get the seconds?

Reply:
updated to 8766 hrs